### ISSUE TITLE
When admin removes group organizer from list, nothing happens.

### ISSUE THREAD

**Issue by dottorer** [lale-help/lale-help#407]
When creating the group initially, the circle admin sets himself as the group organizer. After other helpers joined, he tries to remove himself from the organizer list of a group. Clicking the remove button, nothing happens...

**Comment by phillipoertel** [comment]
@dottorer I assume you mean the other helpers joined as work group organizers (you can't remove the last organizer of a WG). I was able to reproduce this bug the first time I tried. I clicked "Remove" and the admin wasn't removed, although there was one more admin in the group. 

The application logs didn't show any internal error or unusual things, so I tried to recreate the error, **and am now unable to do so**, despite investigating for an hour! The admin is always removed as expected. Can you reliably reproduce this case?

**Comment by dottorer** [comment]
@phillipoertel  I just in DEMO the admin assigned user4@lale.help as organizer of group Betreuung. It also had userm6@lale.help as organizer. Then I logged in as user4 and tried to remove myself as group organizer. Nothing happened on first click. Hover, on second click it said 'user does not have permission to update object' or something of that kind which tells me we removed user4 as organizer, but did not refresh the display.
